### ISSUE TITLE
Any team support?

### ISSUE THREAD

**Issue by hiasl** [authpass/authpass#61]
Servus Herbert!
Any chance your tool becomes team-capable? Currently we are using a commercial tool to share credentials in our team. Any chance this will be possible with authpass?
lg Matthias

**Comment by hpoul** [comment]
@hiasl it depends.. what do you expect from "team-capable" :-) the app already supports opening multiple files as well as sharing through google drive or dropbox.. so nothing stops you from creating a shared "team" kdbx file where everyone knows the passphrase (or has the keyfile). and users can still have their own personal key file which is not shared, but private.. 

the obvious problem is how to exclude people from accessing the passwords once they leave the team.. not sure how to prevent this, if you give people access to a key store they could always just copy all passwords anyway..

**Comment by hiasl** [comment]
I mean 
- separate passphrases for each user
- permissions on password groups
- recovery processes for somehow lost accounts
- removing permissions in case you need it
Of course you can always copy passwords, but it's still more comfortable with the mechanisms described above, but I guess this is causes by the concept of kdbx...right?

**Comment by hpoul** [comment]
yeah, without a server component it would be quite hard, I guess it would be possible to create a kdbx wrapper format, so every user has a different passphrase.. and make it possible to remove access to single users (as long as the user hasn't copied the whole file already).

the most realistic way to achieve something similar (for AuthPass) would be to create a kdbx file for every "password group" .. and have a custom server which manages access permissions and provides the user with the necessary kdbx file and keyfile.. and the user has to provide his personal password for decrypting in addition to the keyfile.. but i'm not sure if this would fully solve your problem :-)

I fear that the only "right" solution would be to have every single password managed by the server.. so it can manage access to each of them and log when which user accesses which passwords. This kind of defeats the kdbx concept, that there is no server which has any knowledge of the passwords 🤷‍♂️️

**Comment by vladaman** [comment]
@hpoul What if there would be some network consensus ? I would love to see team support feature somehow. Whoever cracks this problem can get big chunk of business users. Because other tools are just commercial and server based (centralized)
